src/loaders/common/loader_helper.py

```diff
@@ -11,11 +11,10 @@
 
 import src.common.storage.collection_and_field_names as names
 from src.common.storage.db_doc_conversions import collection_data_id_key
-from src.loaders.common.loader_common_names import (
-    DOCKER_HOST,
-    KB_AUTH_TOKEN,
-    SOURCE_METADATA_FILE_KEYS,
-)
+from src.loaders.common.loader_common_names import (DOCKER_HOST, ERROR, FILE,
+                                                    KB_AUTH_TOKEN,
+                                                    SOURCE_METADATA_FILE_KEYS,
+                                                    STACKTRACE, TOOL)
```

Comment:
The top way of doing the imports is more readable, 1 per line

Reply:
Will keep this in mind in future. 

But moved `create_fatal_dict_doc` and `create_global_fatal_dict_doc` to `tool_common.py` file.

Reply:
Why not just fix now? It'll take 5s

Reply:
Hmm, re the function locations, if functions are shared between the parser and the tools they should go in a file common to both of them. Functions that are for use in tool implementations only should go in tool_common. Whether you have to add imports or not shouldn't be a factor - the criteria is where a new programmer coming into the project would expect to find the functions

Reply:
Makes sense. I will update files accordingly!

Reply:
Updated!

Reply:
:+1: